mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -11,9 +18,29 @@ func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
-func ipv6AddressforDnsDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
-	oldIp := net.ParseIP(old)
-	newIp := net.ParseIP(new)
+func ipv6AddressforDnsDiffSuppress(x, y []string, _ *schema.ResourceData) bool {
+	if len(x) != len(y) {
+		return false
+	}
 
-	return oldIp.Equal(newIp)
+	// compare two lists of unordwered IPs
+	diff := make(map[string]int, len(x))
+	for _, _x := range x {
+		// + parse ipv6 to a key from one list
+		diff[net.ParseIP(_x).String()]++
+	}
+	for _, _y := range y {
+		// + parse ipv6 to a key from the other list
+		if _, ok := diff[net.ParseIP(_y).String()]; !ok {
+			return false
+		}
+		diff[_y] -= 1
```

Comment:
you shouldn't have to worry about the count of the IPs, since DNS records can't contain duplicates I believe.

Reply:
Not dup but added or removed IPs?

Reply:
I was referring to the use of a map[string]int. You really just need map[string]bool or the like, and not worry about incrementing / decrementing since the exact count of IPs don't need to match.